src/wasm/wasm-emscripten.cpp

```diff
@@ -921,11 +992,23 @@ void AsmConstWalker::addImports() {
     }
     wasm.table.initial.addr += queuedTableEntries.size();
   }
+
+  for (const auto& entry : globalImports) {
+    auto* import = new Global;
+    import->name = entry.second;
+    import->module = "GOT.func";
```

Comment:
It's not immediately clear why all the entries in globalImports are always functions. I'm not sure if renaming globalImports to globalFuncImports makes it more clear?

Reply:
Changed to `globalFuncImports`.